GWAS/LMM.ipynb

```diff
@@ -2226,7 +2226,7 @@
     "        topgene<-data[data$P<=${plim},]    \n",
     "    }  \n",
     "    if (${1 if k !=\"\" else 0} ){\n",
-    "        topgene<-data[order(data$P),][1:${k},]    \n",
+    "        topgene<-data[order(data$P),][1:as.double(${k}),]    \n",
```

Comment:
It does not make sense to use double for an index. Why would you do that? What's wrong with using 1:${k}?

Reply:
@gaow it was giving this error and @UxxUnet suggested to do this and it worked after changing it
```
Error: unexpected ',' in:
"if (0 ){
    topgene<-data[order(data$P),][1:,"
Execution halted
Error: unexpected ',' in:
"if (0 ){
    topgene<-data[order(data$P),][1:,"
Execution halted
Error: unexpected ',' in:
"if (0 ){
    topgene<-data[order(data$P),][1:,"
Execution halted
Error: unexpected ',' in:
"if (0 ){
    topgene<-data[order(data$P),][1:,"
Execution halted
```